app/controllers/album.js

```diff
@@ -0,0 +1,18 @@
+const { getAlbums, getAlbumPhotos } = require('../services/album'),
+  logger = require('../logger');
+
+exports.getAlbums = (req, res) =>
+  getAlbums()
+    .then(response => res.status(response.statusCode).send(response.body))
+    .catch(err => {
+      logger.error(`Error getting albums: ${err}`);
+      return res.status(500).send(err);
```

Comment:
shouldnt you send the error you get from the API instead of always returning 500? also use the errors from the `errors.js` file

Reply:
if the request is made i return the message that it returns, whatever that is, if anything else fails i return 500